packages/arb-ts/src/lib/dataEntities/arbTransaction.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  TransactionReceipt,
+  TransactionResponse,
+} from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+
+/**
+ * Eth transaction response with additional arbitrum specific fields
+ */
+export interface ArbTransactionResponse extends TransactionResponse {
+  /**
+   * The sequenced number for this transaction
+   */
+  l1SequenceNumber: BigNumber
+
+  // These fields are commented out for now
+  // we can add them in as we need them and are able to document
+  // them properly
+  // parentRequestId: string
+  // indexInParent: number
+  // arbType: number
+  // arbSubType: number
+
+  /**
+   * The l1 block number that would be used for block.number calls
+   * that occur within this transaction.
+   * See https://developer.offchainlabs.com/docs/time_in_arbitrum
+   */
+  l1BlockNumber: number
+}
+
+export interface FeeStat {
```

Comment:
I think this name is confusing, I'd instead call it FeeComponents, or ResourcesCharged, idk
But FeeStats and FeeStat right now is confusing

Reply:
Updated to FeeStatComponents